corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -274,3 +274,9 @@
     padding-left: 0px !important;
   }
 }
+
+.question-tile-row .gr-no-children.gr-has-no-questions.gr-has-no-nested-questions {
```

Comment:
Is it intentional this only applies to elements that have all three `gr` classes, not each of them individually? It seems _very_ specific. It might also be nicer to nest this under the `.question-tile-row` selector that's already here a couple lines up, like:
```css
.question-tile-row {
  .gr {...}
  .gr-no-children.gr-has-no-questions.gr-has-no-nested-questions {...}
}
```